pyveda/vedaset/stream/vedastream.py

```diff
@@ -146,6 +146,20 @@ def __init__(self, mltype, classes, count, gen, partition, bufsize,
         if auto_startup:
             self._start_consumer()
 
+    @classmethod
+    def from_vc(cls, vc, count=None, bufsize=50, cachetype=collections.deque,
+                auto_startup=False, auto_shutdown=False, fetcher=None, loop=None):
+        if count == None:
+            count = vc.count
+        if count > vc.count:
+            raise TypeError('count must be less than or equal to vc.count')
```

Comment:
No `ValueError` ?  We wanna capitalize n all that stuff in our messages, make them look official like we're a real thing n all.  We want the message to be useful for debugging too, so in a traceback it's nice to know what the args refer to and whatnot, eg: "Input argument 'count' exceeds total size of input VedaCollection instance"

Reply:
I like your less than or equal to though